src/include/verbose.h

```diff
@@ -190,6 +214,25 @@ struct __wt_verbose_multi_category {
 #define __wt_verbose_info(session, category, fmt, ...) \
     __wt_verbose_level(session, category, WT_VERBOSE_INFO, fmt, __VA_ARGS__)
 
+/*
+ * __wt_verbose_info_id --
+ *     Wrapper to __wt_verbose_info_id defaulting the verbosity level to WT_VERBOSE_INFO with a log
+ *     id.
+ */
+#define __wt_verbose_info_id(session, verb_category, log_id, fmt, ...)                   \
+    do {                                                                                 \
+        WT_VERBOSE_MESSAGE_INFO verb_message_info = {                                    \
+          .id = log_id,              /* Unique ID for this specific message */           \
+          .category = verb_category, /* The category of the message */                   \
+          .level = WT_VERBOSE_INFO   /* The verbosity level */                           \
```

Comment:
The definition already indicates what those fields are meant to be.
```suggestion
          .id = log_id,           \
          .category = verb_category,                   \
          .level = WT_VERBOSE_INFO                          \
```

Reply:
Fixed in 0f136343cc588b81524565f6e764cc7d7311d112